environments/production/inventory.ini

```diff
@@ -188,6 +188,32 @@ hqes7.internal-va.commcarehq.org elasticsearch_node_name=hqes7
 [hqesmaster0]
 172.24.32.67 hostname=hqesmaster0 elasticsearch_node_name=hqesmaster0
 
+[es0]
+10.202.41.22 hostname=es0-production elasticsearch_node_name=es0 ec2=yes
```

Comment:
any reason not to follow the `hqes` convention?

Reply:
This came from a discussion here https://github.com/dimagi/commcare-cloud/pull/1894#discussion_r200327569, where we decided to go with these simpler names (`es0`, `web0`, etc.) for the groups, and `{group}-{env}` for the hostnames. We're basically using the fact that we're going to recreate every machine in the environment for the AWS migration to also switch to this naming convention.

Reply:
great, thanks for the explanation.